pkg/kubelet/rkt/rkt.go

```diff
@@ -196,6 +196,15 @@ func (r *Runtime) buildCommand(args ...string) *exec.Cmd {
 	return cmd
 }
 
+// convertToACName converts a string into ACName.
+func convertToACName(name string) appctypes.ACName {
+	// Ignore the error. If there is an error (it should not
+	// happen according to the naming rules), the string will
+	// be empty, MustACName will panic.
```

Comment:
@yujuhong The reason here is I don't like to return an error as well because it's too verbose.  I am pretty sure in real situation, invalid names are already prevented by kubectl when validating the spec (it must match [DNS_LABEL](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L48)  `[a-z0-9]([-a-z0-9]*[a-z0-9])?)`), so it's not gonna panic.